netctl/netctl.go

```diff
@@ -33,12 +33,12 @@ func createPolicy(ctx *cli.Context) {
 	tenant := ctx.String("tenant")
 	policy := ctx.Args()[0]
 
-	logrus.Infof("Creating policy %s:%s", tenant, policy)
-
 	errCheck(ctx, getClient(ctx).PolicyPost(&contivClient.Policy{
 		PolicyName: policy,
 		TenantName: tenant,
 	}))
+
+	fmt.Printf("Creating policy %s:%s\n", tenant, policy)
```

Comment:
The reason i kept it fmt.Printf is that it does not say[INFO] like logrus.Infof, it just displays the message of the console. Since its better to just display the message on the console for the user, beautified it.